R/language.R

```diff
@@ -33,10 +49,16 @@
 #' is_atomic_lang(quote(call()))
 #'
 #' # With the exception of names, atomic language objects have the
-#' # property of being self-quoting:
+#' # property of being self-quoting literals:
```

Comment:
Maybe we should make it `is_literal()`? 

Then a node of the parse tree is either a literal, a name, or a call. (Or a pairlist in the special case of function arguments)